CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 * Your contribution here.
 
+* [#1167](https://github.com/ruby-grape/grape/pull/1167): Refactor and extend coercion and type validation system - [@dslh](https://github.com/dslh).
```

Comment:
Maybe should be added to changelog as a line, like that we support `File` now out of the box.